Assets/MixedRealityToolkit/_Core/Definitions/Utilities/Handedness.cs

```diff
@@ -30,6 +30,11 @@ public enum Handedness : byte
         /// </summary>
         Both = Left | Right,
         /// <summary>
+        /// Global catchall, used to map actions to any controller (provided the controller supports it)
+        /// *Note, by default the specific hand actions will override settings mapped as both
+        /// </summary>
+        Any = None | Left | Right | Other,
```

Comment:
Not sure if this is needed.

Reply:
I like catch-alls and also good if I want to configure just one mapping for All controllers / SDK types